static/src/js/components/CollectionDetails/CollectionDetailsCloudAccessInstance.scss

```diff
@@ -0,0 +1,92 @@
+.collection-details-cloud-access-direct-distribution-information {
+    &__info {
+        > dt {
+            font-size: 0.75rem;
+            font-weight: 400;
+        }
+        > dd {
+            font-size: 0.925rem;
+            font-weight: 700;
+            word-wrap: break-word;
+        }
+    }
+
+    &__cloud-access-content {
+    border: 1px solid $color__black--200;
+    padding: 1.25rem
+  }
+
+  &__cloud-access-title {
+    border-top: 1px solid $color__black--200;
+    padding-top: 1.25rem;
+  }
+
+  &__link {
+    display: block;
+    padding: 0;
+    white-space: nowrap;
+    font-weight: 700;
+    font-size: .925rem;
+    line-height: 1.5rem;
+    text-decoration: none;
+
+    &:hover {
+      text-decoration: underline !important;
+    }
+
+    .direct-distribution-information__links--horizontal & {
+      display: inline-block;
+      vertical-align: baseline;
+
+      &:hover {
+        text-decoration: underline;
+      }
+
+      &:last-child {
+        padding-top: $spacer/8;
+      }
+
+      .button {
+        display: inline-block;
+
+        &__contents {
+          vertical-align: baseline;
+        }
+      }
+    }
+  }
+}
+
+.collection-details-cloud-access-variable {
+    &__direct-distribution-information-list {
+      list-style-type: none;
+      padding-left: .5rem;
+    }
+
+    &__list-item {
+      overflow-y: hidden;
+      list-style-position: inside;
+      margin-left: 10px;
+    }
+
+    &__definition-list-container{
+      margin-left: 1rem;
+    }
+    
+    &__instance-field-title {
+      font-weight: bold;  
+      display:inline;
+    }
+
+    &__instance-field-list-title {
+      font-weight: bold;
+      display:inline;
+      &:before {
+        content: "• "
```

Comment:
This feels off to me, if we want a bulleted list just let the list element do it

Reply:
Yea I tried doing this for a long time and was getting quite a few issues with spacing. I did see we were using this "•" char elsewhere in the code

Reply:
This is a really good catch. We definitely want to use the default unordered list here.